contrib/ci/build_windows.sh

```diff
@@ -39,14 +42,23 @@ meson .. \
     -Dlibxmlb:gtkdoc=false \
     -Dlibjcat:man=false \
     -Dlibjcat:gpg=false \
+    -Dlibjcat:tests=false \
     -Dlibjcat:introspection=false \
     -Dgusb:tests=false \
     -Dgusb:docs=false \
     -Dgusb:introspection=false \
     -Dgusb:vapi=false $@
 VERSION=$(meson introspect . --projectinfo | jq -r .version)
-ninja -v
-ninja -v install
+
+# run tests
+export WINEPATH="/usr/x86_64-w64-mingw32/sys-root/mingw/bin/;$build/libfwupd/;$build/libfwupdplugin/;$build/subprojects/libxmlb/src/;$build/subprojects/gcab/libgcab/;$build/subprojects/libjcat/libjcat/;$build/subprojects/gusb/gusb/"
+ninja -C $build -v
+ninja -C $build test
+
+# switch to release optimisations
+meson configure -Dtests=false -Dbuildtype=release
```

Comment:
do any other subprojects need tests turned off too for this?

Reply:
I think we turn them off for the "main" configure step already.